content/configuration/webhooks.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Webhooks
+description: Set up webhooks for automatic builds
+weight: 6
+aliases: /building/webhooks
```

Comment:
this needs the previous alias too (from /flutter-configuration/webhooks)

Reply:
The alias actually already pointed to here.